hedera-node/src/main/java/com/hedera/services/store/TypedTokenStore.java

```diff
@@ -149,13 +156,10 @@ public long currentMintedNfts() {
 	 * in order for its changes to be applied to the Swirlds state, and included in the
 	 * {@link com.hedera.services.state.submerkle.ExpirableTxnRecord} for the active transaction.
 	 *
-	 * @param token
-	 * 		the token in the relationship to load
-	 * @param account
-	 * 		the account in the relationship to load
```

Comment:
nit: I think that those changes for the javadoc are auto formatted by a tool. Lets revert those changes and ask in the weekly sync the tool that they use for the java doc